.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please provide the following details along with the issue description - Thank you for the contribution.
+
+* `node` version is: `node --version` would log the version
+* `npm` version is: `npm --version` would log the version
+* `webpack` version is: the value next to `version` key at `./node_modules/webpack/package.json`
+* `webpack-validator` version is: the value next to `version` key at `./node_modules/webpack-validator/package.json`
+* `OS` details: http://whatsmyos.com/ would provide the details
+* error log:
+
+Any other details, that might be helpful to reproduce the issue.
```

Comment:
So this template refers to bug reports, but issues are also for features and ideas right? So I'd think that this might have to be formulated more generally?